Userland/Applications/PixelPaint/CreateNewGuideDialog.gml

```diff
@@ -0,0 +1,72 @@
+@GUI::Frame {
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+    }
+
+    @GUI::Widget {
+        layout: @GUI::VerticalBoxLayout {
+            margins: [4,4,4,4]
+        }
+
+        @GUI::GroupBox {
+            title: "Orientation"
+            //fixed_height: 25
```

Comment:
Not sure if the "no commented code" applies here, but please either remove this, or leave a comment next to it stating why it's commented : )